src/main/java/com/docker/jersey/restapi/DockerRegistrationApi.java

```diff
@@ -0,0 +1,31 @@
+package com.docker.jersey.restapi;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import java.util.Optional;
+
+@Path("/dockerRegistration")
+public class DockerRegistrationApi {
+    @POST
+    public Response createImage() {
```

Comment:
POST method should receive a body of a specific type. So create a package com.docker.jersey.model
Create a class called DockerParameters and apply to this method.